pyverbs/libibverbs.pxd

```diff
@@ -723,7 +723,8 @@ cdef extern from 'infiniband/verbs.h':
     int ibv_query_rt_values_ex(ibv_context *context, ibv_values_ex *values)
     int ibv_get_async_event(ibv_context *context, ibv_async_event *event)
     void ibv_ack_async_event(ibv_async_event *event)
-
+    int ibv_fork_init()
+    e.ibv_fork_status ibv_is_fork_initialized()
```

Comment:
remove the "e." after removing the enums cimport

Reply:
Done